### ISSUE TITLE
Move FAQ link to somewhere it's findable

### ISSUE THREAD

**Issue by amyrhoda** [swcarpentry/DEPRECATED-website#200]
There is currently no link to the FAQ that's easy to find. We need to move it to the front page.

(This is dependent on completing Issue #196 but I don't know how to define dependencies in Git yet.)

**Comment by wking** [comment]
On Tue, Jan 08, 2013 at 11:00:46AM -0800, Amy Brown wrote:

> (This is dependent on completing Issue #196 but I don't know how to
> define dependencies in Git yet.)

GitHub issues don't have explicit dependency metadata.  You just make
a comment (like you did), and hope that interested parties will read
your comment.

**Comment by gvwilson** [comment]
Closed by e558b72.

**Comment by wking** [comment]
On Tue, Jan 08, 2013 at 12:55:00PM -0800, Greg Wilson wrote:

> Closed by e558b72.

Definately not required, but `hub` makes it easy to attach pull
requests to existing issues [1](https://github.com/defunkt/hub#git-pull-request):

```
$ git checkout -b faq
…hack hack hack…
$ git push -u gvwilson faq
$ hub pull-request -i 200
```
